dropwizard-async-http-client/src/main/java/smartthings/dw/asynchttpclient/AsyncHttpClientConfig.java

```diff
@@ -0,0 +1,16 @@
+package smartthings.dw.asynchttpclient;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AsyncHttpClientConfig {
+	private Map<String, Object> properties = new HashMap<>();
```

Comment:
here are the defaults: https://github.com/AsyncHttpClient/async-http-client/blob/master/client/src/main/resources/ahc-default.properties